.pre-commit-config.yaml

```diff
@@ -16,4 +16,8 @@ repos:
   rev: 6.0.0
   hooks:
   - id: flake8
-    files: "advisor_listener|common|database|listener|evaluator|manager|taskomatic|tests|vmaas_sync|notificator"
+    files: "common|database|listener|evaluator|manager|taskomatic|tests|vmaas_sync|notificator"
+- repo: https://github.com/PyCQA/isort
+  rev: 5.12.0
+  hooks:
+  - id: isort
```

Comment:
In QA we've been using https://github.com/asottile/reorder-python-imports instead of `isort`, it's written by a pre-commit and pytest developer and it is  `Similar to isort but uses static analysis more.`

Reply:
otherwise LGTM

Reply:
okay, I will swap for this one since you have some experience with it